helphours/static/scripts/join.js

```diff
@@ -0,0 +1,31 @@
+// How often we check if the queue is open
+var UPDATE_INTERVAL_SECONDS = 5;
```

Comment:
Shouldn't this be a constant instead of a variable?

```suggestion
const UPDATE_INTERVAL_SECONDS = 5;
```

Reply:
Yeah you're right, halfway through writing the JavaScript I gave up on trying to support stuff before ES6 lol

Reply:
If it becomes that much of an issue, then we can always run this through Babel to compile down to ES5 anyway.  Given that we're using the `fetch` API, I think that going all-in on ES6 makes the most sense.